libraries/AP_IOMCU/AP_IOMCU.cpp

```diff
@@ -117,7 +118,7 @@ void AP_IOMCU::thread_main(void)
     uart.begin(1500*1000, 128, 128);
     uart.set_unbuffered_writes(true);
 
-#if HAL_WITH_IO_MCU_BIDIR_DSHOT
+#if HAVE_AP_BLHELI_SUPPORT && HAL_WITH_IO_MCU_BIDIR_DSHOT
```

Comment:
I'd have to look more closely but I don't think this is correct - I don't think we compile blheli into iomcu

Reply:
AP_IOMCU does use AP_BLHeli, and that is the issue. It doesn't build with sim-on-hw